Xamarin.Forms.Core/Internals/ResourceLoader.cs

```diff
@@ -5,8 +5,17 @@ namespace Xamarin.Forms.Internals
 {
 	public static class ResourceLoader
 	{
+		static Func<AssemblyName, string, string> resourceProvider;
+
 		//takes a resource path, returns string content
-		public static Func<AssemblyName, string, string> ResourceProvider { get; internal set; }
+		public static Func<AssemblyName, string, string> ResourceProvider {
+			get => resourceProvider;
+			internal set {
+				DesignMode.DesignModeEnabled = value != null;
```

Comment:
This was deliberately set indiscriminately to `true` as I felt it would be worse for this property to flip-flop between `true` and `false` depending on whether the previewer nulls out the old delegate after the render completes, or if it leaves it in place until the next delegate is set.

Reply:
fixed